test/PowerShellEditorServices.Test.E2E/TestsFixture.cs

```diff
@@ -70,6 +71,11 @@ public async Task InitializeAsync()
                 "-Stdio"
             };
 
+            if (IsDebugAdapterTests)
```

Comment:
If this is intended for a particular subclass, ideally the superclass shouldn't know about it at all. But in test classes, not such a big deal

Reply:
yeah this isn't final. There's more work I need to do wrt testing.